src/parser.ts

```diff
@@ -0,0 +1,106 @@
+import { Flags } from '@oclif/core';
+import { Parser } from '@asyncapi/parser/cjs';
+import { AvroSchemaParser } from '@asyncapi/parser/cjs/schema-parser/avro-schema-parser';
+import { OpenAPISchemaParser } from '@asyncapi/parser/cjs/schema-parser/openapi-schema-parser';
+import { RamlSchemaParser } from '@asyncapi/parser/cjs/schema-parser/raml-schema-parser';
+import { getDiagnosticSeverity } from '@stoplight/spectral-core';
+import { html, json, junit, stylish, teamcity, text, pretty } from '@stoplight/spectral-cli/dist/formatters';
+import { OutputFormat } from '@stoplight/spectral-cli/dist/services/config';
```

Comment:
it is pretty risky to import these helpers from `dist` of the CLI.....breaking changes may hurt us later

Reply:
It will give you errors during the CLI build that these exports don't exist in the path. There is no other way, and creating these functions from scratch or copying them to our codebase is pointless. There is no good solution here and this is the least invasive.

Reply:
then we should at least make sure that in package.json we do not allow `^` for spectral-cli dependency